lib/auth.js

```diff
@@ -76,22 +103,25 @@ const passwordTests = {
     options: { number: 1 },
   },
   MIN_NUMBERS: {
-    test: (password, options) => {
-      const NUMBERS_RANGE = [[49, 57]];
-      return stringIncludesChars(password, NUMBERS_RANGE, options.number);
-    },
+    test: (password, options) =>
+      stringIncludesChars(password, NUMBERS_RANGE, options.number),
     hint: options => ({ name: 'MIN_NUMBERS', number: options.number }),
     options: { number: 1 },
   },
   MIN_SPECIAL_CHARS: {
-    test: (password, options) => {
-      const SPECIAL_CHARS_RANGE = [[32, 47], [58, 64], [91, 96], [123, 126]];
-      // https://www.owasp.org/index.php/Password_special_characters
-      return stringIncludesChars(password, SPECIAL_CHARS_RANGE, options.number);
-    },
+    test: (password, options) =>
+      stringIncludesChars(password, SPECIAL_CHARS_RANGE, options.number),
     hint: options => ({ name: 'MIN_SPECIAL_CHARS', number: options.number }),
     options: { number: 1 },
   },
+  FOUND_TOPOLOGY: {
+    test: password => isTopology(password),
```

Comment:
I think we should allow to pass custom list of topologies (but still have our default `passwordTopologies`).